circuitpython/CIRC00/neopixel_builtin.py

```diff
@@ -1,13 +1,13 @@
-# CIRC00 -  Metro Express NeoPixel
-# (CircuitPython)
-# this circuit was designed for use with the Metro Express Explorers Guide on Learn.Adafruit.com
+"""
+'neopixel_builtin.py'.
 
-# by Asher Lieber for Adafruit Industries.
+=================================================
+control a LED with an IR Remote
+"""
 
-# import required libraries
-import neopixel
-import board
 import time
+import board
+import neopixel
 
 # we only have one neopixel on our metro express
 neo = neopixel.NeoPixel(board.NEOPIXEL, 1)
```

Comment:
You may want to consider adding `brightness=0.5` or another line of `neo.brightness = 0.5` or some other number that is less than 1. As is, this is super bright.